content/design-systems-for-developers/react/en/review.md

```diff
@@ -141,23 +163,21 @@ git commit -am “make Button pop”
 git push -u origin improve-button
 ```
 
-Navigate to GitHub.com and open up a pull request for the `improve-button` branch.
-
-![Creating a PR in GitHub](/design-systems-for-developers/github-create-pr.png)
+Navigate to GitHub.com and open a pull request for the `improve-button` branch. Once opened, the CI job to publish Storybook will run.
 
-![Created a PR in GitHub](/design-systems-for-developers/github-created-pr.png)
+![Created a PR in GitHub](/design-systems-for-developers/github-created-pr-actions.png)
 
-Go to the Netlify URL in your PR checks to find your button component.
+In your list of PR checks at the bottom of the page, click **Storybook Publish** to view the published Storybook with the new changes.
 
-![Button component changed in deployed site](/design-systems-for-developers/netlify-deployed-site-with-changed-button.png)
+![Button component changed in deployed site](/design-systems-for-developers/chromatic-deployed-site-with-changed-button.png)
 
 For each component and story that changed, copy the URL from the browser address bar and paste it wherever your team manages tasks (GitHub, Asana, Jira, etc) to help teammates quickly review the relevant stories.
```

Comment:
@domyen / @jonniebigodes do we think we should talk about Chromatic's UI review feature at this point, if only because otherwise people may be confused about the fact that we are describing a workflow here that is different to that, and thus we will confuse them?

Maybe at this point we should just add an aside:

> Chromatic also offers a complete UI Review workflow built into the product as part of its paid offering. The technique of copying Storybook links into a GitHub PR works at a smaller scale (and with any service that hosts your Storybook, not just Chromatic), but as your use increases you may consider that services as it automates the process.